third-party/snyk/snapshot.json

```diff
@@ -101,7 +101,7 @@
       {"id":"npm:knockout:20130701","severity":"medium","semver":{"vulnerable":[">=2.1.0-pre <3.0.0"]}}
     ],
     "lodash":[
-      {"id":"SNYK-JS-LODASH-567746","severity":"medium","semver":{"vulnerable":["<=4.17.15"]}},
+      {"id":"SNYK-JS-LODASH-567746","severity":"medium","semver":{"vulnerable":["<4.17.16"]}},
```

Comment:
noop ???? there's no versions between these two https://www.npmjs.com/package/lodash

Reply:
Hey @connorjclark - there indeed is no version there, but this has the added bonus of telling a user that a fixed version exists, and even stating it. The old variant would not have told you which version to upgrade to (the next version might have been `5.0.0` for all the the user knows).